Arrowgene.Ddon.Cli/Command/DbMigrationCommand.cs

```diff
@@ -0,0 +1,95 @@
+using Arrowgene.Ddon.Database;
+using Arrowgene.Ddon.Database.Model;
+using Arrowgene.Ddon.Shared;
+using Arrowgene.Logging;
+using System;
+using System.IO;
+
+namespace Arrowgene.Ddon.Cli.Command
+{
+    public class DbMigrationCommand : ICommand
+    {
+        private static readonly ILogger Logger = LogProvider.Logger<Logger>(typeof(ServerCommand));
+
+        public string Key => "dbmigration";
+
+        public string Description => $"Commandline tool to update the database\n\n" +
+                                     $"usage:\n" +
+                                     $"{Key} <previous version>\n" +
+                                     $"The previous version argument is required only when using sqlite as the database engine.";
+
+        public CommandResultType Run(CommandParameter parameter)
+        {
+           
+            string settingPath = Path.Combine(Util.ExecutingDirectory(), "Files/Arrowgene.Ddon.config.json");
+            Setting settings = Setting.Load(settingPath);
+            if (settings == null)
+            {
+                return CommandResultType.Exit;
+            }
+
+            DatabaseSetting dbSettings = settings.DatabaseSetting;
+            if (dbSettings == null)
+            {
+                return CommandResultType.Exit;
+            }
+
+            Enum.TryParse(dbSettings.Type, true, out DatabaseType dbType);
```

Comment:
We do we physically need to make a new db file when updating sqlite?

Reply:
There's no _need_ but it's what was implied in the existing code with the Version variable, where the sqlite file name would be decided by it. It also serves as a backup, in a way

Reply:
Seems weird we provide a backup for sqlite but not other database types.

Reply:
We could ditch the versioning from the file name directly, remove the version number from the suffix and operate in the same file